vncviewer/DesktopWindow.cxx

```diff
@@ -51,6 +51,8 @@
 
 #ifdef __APPLE__
 #include "cocoa.h"
+#include <CoreGraphics/CGGeometry.h>
+#include <CoreGraphics/CGRemoteOperation.h>
```

Comment:
@CendioOssman did you get this working on macOS? or are there any further tweaks needed?

Reply:
Still gettin `fatal error: CoreGraphics/CoreGraphics.h: No such file or directory`. So I would still suggest the top Carbon header.

Reply:
how do you do that? i don't have a way to test these changes, so i'm flying blind.

Reply:
Just change it to `#include <Carbon/Carbon.h>` :)